temporian/core/serialize.py

```diff
@@ -29,7 +29,10 @@
 
 
 def save(
-    inputs: Dict[str, Event], outputs: Dict[str, Event], path: str
+    inputs: Optional[processor.MultipleEventArg],
+    outputs: processor.MultipleEventArg,
+    path: str,
+    infer_inputs: bool = False,

```

Comment:
Propose defaulting to True (at least when user doesn't specify `inputs`)

Reply:
*Remove infer_inputs arguments. When inputs=None, the input is inferred automatically.